### PR TITLE
Support array result include sequence action

### PR DESCRIPTION
**Pull request by ningyougang**
Depend on below prs: 
- https://github.com/apache/openwhisk/pull/5290
- https://github.com/apache/openwhisk-runtime-go/pull/170

- [x] Support array result
  - [x] make swift runtime to support array result for common action
  - [x] make swift runtime to support array result for sequence action (support array as input param)

### CODE REVIEW

**Comment by ningyougang** [comment]
The steps of test support array result for sequence action in swift(4.2, 5.1,  5.3)
* Write actions
```swift
[root@nccddev130026 ~]# cat ~/split.swift 
func main(args: Any) -> Any {
    // don't do real split operation, just return array for next action's input param
    return ["e1", "e2"]
}

[root@nccddev130026 ~]# cat ~/sort.swift 
func main(args: Any) -> Any {
    // don't do real sort operation, just return the array direclty
    return args
}
```
* Create actions and invoke it
```shell
wsk -i action create /whisk.system/utils/split-swift --kind swift:5.3 ~/split.swift
wsk -i action create /whisk.system/utils/sort-swift --kind swift:5.3 ~/sort.swift
wsk -i action create mySequence-swift --sequence /whisk.system/utils/split-swift,/whisk.system/utils/sort-swift
wsk -i action invoke --result mySequence-swift -r -v
```

File: README.md [hunk 1]
```diff
@@ -33,15 +33,38 @@ The traditional support for the dictionary still works:
 ```swift
 import Foundation
 
-func main(args: [String:Any]) -> [String:Any] {
-    if let name = args["name"] as? String {
+func main(args: Any) -> Any {
```
Comment (ningyougang) [inline README.md:36]: In order to support array result, from the new main signature. we can see, the the input param and outout param should be changed to `Any` both.

File: core/swift51Action/Dockerfile [hunk 2]
```diff
@@ -31,12 +31,12 @@ RUN curl -sL \
...
   | tar xzf -\
   && cd openwhisk-runtime-go-*/main\
-  && GO111MODULE=on go build -o /bin/proxy
+  && GO111MODULE=on CGO_ENABLED=0 go build -o /bin/proxy
 
 FROM swift:5.1.5
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=release
+ARG GO_PROXY_BUILD_FROM=source
```
Comment (ningyougang) [inline core/swift51Action/Dockerfile:39]: Need to change it to `release` due to https://github.com/apache/openwhisk-runtime-java/pull/140#discussion_r939689436

Comment (ningyougang) [inline core/swift51Action/Dockerfile:39]: Need to change it to `release` due to https://github.com/apache/openwhisk-runtime-java/pull/140#discussion_r939689436

Comment (ningyougang) [reply core/swift51Action/Dockerfile]: Already changed to `release`

**Review by dgrove-oss — APPROVED** [review]
LGTM

**Pull request merged**